test/visual/index.test.ts

```diff
@@ -1,11 +1,14 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
+import path from "path";
 import { ScreenshotPageObject } from "@cloudscape-design/browser-test-tools/page-objects";
 import { expect, test } from "vitest";
-import { routes } from "../../pages/pages";
```

Comment:
Removed import across sub-projects to prevent e2e tests importing real pages in tests scope